packages/validator-bonds-sdk/__tests__/bankrun/initSettlement.spec.ts

```diff
@@ -22,6 +27,13 @@ import { Keypair, PublicKey } from '@solana/web3.js'
 import { createVoteAccount } from '../utils/staking'
 import { verifyError } from '@marinade.finance/anchor-common'
 import { initBankrunTest } from './bankrun'
+import { isInitialized } from '../../src/settlementClaims'
+
+const tenKilobytes = 10 * 1024
+// number of bytes in 10MB * 8; 56 bytes is size for account header
+// as we know there is 56 bytes for account header it fits 8 times more items
```

Comment:
8 times _less_? But this comment really took some time to digest...

maxMerkleNodesAccountSize is in bits? So maybe it would help to rename to maxMerkleNodesAccountCount? wdyt?